src/main/java/com/capitalone/dashboard/model/AuditType.java

```diff
@@ -9,6 +9,7 @@
     STATIC_SECURITY_ANALYSIS,
     LIBRARY_POLICY,
     TEST_RESULT,
+    FEATURE_TEST,
```

Comment:
It's a separate type because in apiaudit there is an "evaluator map" that maps the incoming audit type to the evaluator file that will do the processing. For now at least we decided on having a separate evaluator for this flow instead of adding conditions and complications to the existing "RegressionTestEvaluator" (which maps to the TEST_RESULT audit type)